cmd/nvidia-container-runtime-hook/container_config.go

```diff
@@ -31,15 +29,11 @@ const (
 	capSysAdmin = "CAP_SYS_ADMIN"
 )
 
-const (
-	deviceListAsVolumeMountsRoot = "/var/run/nvidia-container-devices"
-)
-
 type nvidiaConfig struct {
-	Devices            string
+	Devices            []string
 	MigConfigDevices   string
 	MigMonitorDevices  string
-	ImexChannels       string
+	ImexChannels       []string
```

Comment:
```suggestion
	IMEXChannels       []string
```

Reply:
I think `Imex` matches how we treat `Mig` in this context.